pkg/store/store.go

```diff
@@ -29,6 +29,20 @@ func LimaDir() (string, error) {
 	return dir, nil
 }
 
+// LimaConfigDir returns the path of the config directory, $LIMA_HOME/_config.
+// It will create the directory if it doesn't yet exist.
+func LimaConfigDir() (string, error) {
+	limaDir, err := LimaDir()
+	if err != nil {
+		return "", err
+	}
+	configDir := filepath.Join(limaDir, "_config")
```

Comment:
“_config” should be a constant.

Reply:
Yes, I was tempted to put it into `store/filenames`, but that package claims it only defines names for the instance directories. So where should `_config` be defined? New package?

Reply:
pkg/store.ConfigDir